app/controllers/pg_hero/home_controller.rb

```diff
@@ -107,12 +122,14 @@ def index_bloat
 
     def live_queries
       @title = "Live Queries"
+      @citus_enabled = @database.citus_enabled?
```

Comment:
Are these changes belong to this PR? Or did github somehow shows the wrong diff.

Reply:
Yes they belong because since now I have added **Citus**  tab on the outer UI if Citus is enabled, `citus_enabled` needs to be part of every tab controller.